apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx

```diff
@@ -164,12 +164,22 @@ export default function StakingSummary({ navigation, route }: Props) {
 
   const onContinue = useCallback(async () => {
     navigation.navigate(ScreenName.NearStakingSelectDevice, {
+      source: route.params.source,
+      validatorName: chosenValidator.validatorAddress,
```

Comment:
Do you have a real validator name here? In LLD we have only a "recipient"???

Reply:
You are right. Updated 👍